packages/nvidia_nat_strands/tests/test_strands_llm.py

```
@@ -0,0 +1,424 @@
+
+    def test_nim_compatible_openai_model_format_request_message_content_other_types(self):
+        """Test NIMCompatibleOpenAIModel.format_request_message_content handles other content types."""
+        # Test that non-reasoningContent types would fall back to parent
+        text_content = {"text": "Hello world"}
+
+        # The method should fall back to parent implementation for non-reasoning content
+        content = text_content
+
+        if "reasoningContent" in content:
+            # This should not be executed for text content
+            assert False, "reasoningContent handling should not be triggered for text content"
```

Ruff (0.14.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)